docs/dev/c4/containers.md

```diff
@@ -0,0 +1,37 @@
+```mermaid
+C4Context
+  title [Container] FloPy
+
+  Boundary(mf6, "MODFLOW 6"){
+    System(src, "Source code",)
```

Comment:
Try to stay within the context of components. Perhaps we can remove the relation to the developer and only call this component something like `simulator` or `data model`.